stock_warehouse_orderpoint_stock_info/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2016 OdooMRP Team
+# Copyright 2016 AvanzOSC
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Warehouse Orderpoint Stock Info",
+    "version": "12.0.1.0.0",
+    "depends": ["stock"],
+    "author": "OdooMRP team, "
+              "AvanzOSC, "
+              "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse",
+    "license": "AGPL-3",
```

Comment:
installable attribute missing

Reply:
I add installable already.
I think it default = True because I try add
installable False -> Can't install
installable True -> Can install
Not installable -> Can install